atomic_reactor/plugins/exit_koji_import.py

```diff
@@ -149,6 +156,47 @@ def get_buildroot(self, worker_metadatas):
 
         return buildroots
 
+    def get_logs(self):
+        """
+        Build list of log files
+
+        :return: list, of log files
+        """
+
+        logs = None
+        output = []
+
+        # Collect logs from server
+        try:
+            logs = self.osbs.get_orchestrator_build_logs(self.build_id)
+        except OsbsException as ex:
+            self.log.error("unable to get build logs: %r", ex)
+            return output
+        except TypeError:
+            # Older osbs-client has no get_orchestrator_build_logs
+            self.log.error("OSBS client does not support get_orchestrator_build_logs")
+            return output
+
+        platform_logs = {}
+        for entry in logs:
+            platform = entry.platform
+            platform_logs.setdefault(platform, [])
+            if platform not in platform_logs.items():
+                filename = 'orchestrator' if platform is None else platform
+                platform_logs[platform] = NamedTemporaryFile(prefix="%s-%s" %
+                                                             (self.build_id, filename),
+                                                             suffix=".log", mode='wb')
```

Comment:
I think this wants to be `mode='wt'`. The interface for `get_orchestrator_build_logs()` tells us that `.line` will be a 'str' ('unicode' in Python 2), not a 'bytes', so in order to write it to a file in binary mode we'd need to encode it (to e.g. UTF-8). Using `mode='wt'` means this happens automatically for us.

Reply:
What's the resolution here?

Reply:
It had to be 'wb', with explicit `.encode()` calls before writing, to make Python 2 happy -- do you know another way?

Reply:
No, I do not. Currently resolution seems good to me.